### DISCUSSION TITLE
Changing Color scheme for the template

### DISCUSSION THREAD

**Discussion by vishwa27yvs** [kevinlin1/just-the-class#47] [Q&A]
Thanks for creating this template! Its quite customizable and easy to use.

I am quite new to Jekyll, and trying to change the color for the website. I made the following change in **_sass/custom/custom.scss**

```
.main-content {
  dl {
    display: block;
    grid-template-columns: none;
  }

  dt {
    font-weight: 700;
    text-align: start;

    &::after {
      content: normal;
    }
  }

  dd {
    font-weight: normal;

    + dt {
      margin-top: 1em;
    }
  }

  h1 {
    color: #8C1515;
  }

  h2 {
    color: #8C1515;
  }

  a {
    color: #8C1515;
  }

  .anchor-heading {
    color: #8C1515;
  }
  
}

a {
  color: #8C1515;
}

svg {
  color: #8C1515;
}
```

I can see that the color changes locally, but not on the hosted site, does anyone have any leads to why this can happen.

**Comment by kevinlin1** [comment]
Hi Vishwa,

If the changes are showing up on your local deployment but not on the hosted site, I recommend the following two tips.

1. Double check that you have committed and pushed the chance to your hosted site. If you're using GitHub Pages, ensure that the commit deploy action was successful.
2. Visit your hosted website and hard-refresh your page to force a cache reload. Asset caching is very conservative so changes can take several days to propagate without a manual refresh.
